pkg/operator/operator.go

```diff
@@ -31,8 +31,8 @@ var deploymentVersionHashKey = operatorv1.GroupName + "/rvs-hash"
 
 const (
 	targetName        = "csi-snapshot-controller"
-	targetNamespace   = "openshift-cluster-storage-controllers"
-	operatorNamespace = "openshift-cluster-storage-operators"
+	targetNamespace   = "openshift-cluster-storage-operator"
+	operatorNamespace = "openshift-cluster-storage-operator"
```

Comment:
Should we consolidate these into a single variable?

Reply:
I'd rather keep them separated, just in case someone wants to separate them again.

Reply:
Sounds good to me.